src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/client/PersonMatchClient.kt

```diff
@@ -3,14 +3,21 @@ package uk.gov.justice.digital.hmpps.personrecord.client
 import org.springframework.cloud.openfeign.FeignClient
 import org.springframework.web.bind.annotation.PostMapping
 import org.springframework.web.bind.annotation.RequestBody
-import uk.gov.justice.digital.hmpps.personrecord.client.model.match.PersonMatchRequest
+import uk.gov.justice.digital.hmpps.personrecord.client.model.match.PersonMatchMigrateRequest
+import uk.gov.justice.digital.hmpps.personrecord.client.model.match.PersonMatchRecord
 
 @FeignClient(
   name = "person-match",
   url = "\${person-match.base-url}",
 )
 interface PersonMatchClient {
 
+  @PostMapping("/person")
+  fun postPerson(@RequestBody personMatchRecord: PersonMatchRecord)
+
+  @PostMapping("/person")
```

Comment:
DeleteMapping? Otherwise how does this differ from the above?
(btw Feign - ugh)